cmd/health-agent/main.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 var (
+	configDir = flag.String("configDir", "/etc/health-agent/tests.d/",
```

Comment:
Please make the default be "/etc/health-agent" and then internally add the "tests.d"